docs/GettingStarted/HelmSetup.md

```diff
@@ -13,6 +13,8 @@ sidebar_position: 3
 
 ## Quick Install
 
+#### You can also check https://merico-dev.github.io/devlake-helm-chart for the easiest way to deploy your devlake
```

Comment:
I think "helm repo" deserves a separate section for itself alone, or it will be easily ignored

Reply:
https://bitnami.com/stack/mysql/helm may be a good example of how to describe a quick install.